lib/sles4sap/ipaddr2.pm

```diff
@@ -584,16 +586,20 @@ sub ipaddr2_internal_key_accept {
         # Try two different variants of the same command.
         $ret = script_run(join(' ',
                 'ssh',
+                '-E /var/tmp/ssh_sut.log',
```

Comment:
Eventually create use constant SSH_LOG => '/var/tmp/ssh_sut.log on the top of the lib/sles4sap/ipaddr2.pm file and use it everywhere